### ISSUE TITLE
"This website requires JavaScript." Does it?

### ISSUE THREAD

**Issue by JonathanWilbur** [go-gitea/gitea#31607]
### Feature Description

The ability to work without using Javascript on the front end. IMO, Javascript should always be a feature, not a requirement.

That said, I tried the demo site with JS disabled and AFAICT, everything seems to work great! Could it be that the "This website requires Javascript" banner is wrong?

**Comment by stuzer05** [comment]
Why disable javascript?

**Comment by delvh** [comment]
It requires JS for everything that renders dynamic content/allows interaction.
While Gitea is still good with what JS it requires, there are certain things you simply don't have without JS such as reactions.

**Comment by JonathanWilbur** [comment]
Would you accept a PR to change the verbiage of the message to something like "Some features may not be available with Javascript disabled?"

**Comment by lunny** [comment]
We don't know how it cannot be used and we have enough resources to list them. So that, I don't think it's better to mention some features work even JavaScript disabled.

**Comment by wxiaoguang** [comment]
Yes, it requires. Otherwise a lot of features do not work. Without JS, you could only view some static content with broken pages.
